pdc/apps/package/filters.py

```diff
@@ -164,3 +164,26 @@ class BuildImageRTTTestsFilter(django_filters.FilterSet):
     class Meta:
         model = models.BuildImage
         fields = ('build_nvr', 'test_result', 'image_format')
+
+
+class ReleasedFilesFilter(django_filters.FilterSet):
+    file             = MultiValueFilter()
+    package          = MultiValueFilter()
+    build            = MultiValueFilter()
+    released_date    = MultiValueFilter()
+    release_date     = MultiValueFilter()
```

Comment:
As this is Date format, we can just use "2017-11-11" to filter, I have test that, works.

Reply:
Yes, but is that useful for anyone to be able to filter by exact date? Also the date format in filter is not verified since it can be any string.

Reply:
What do you mean of exact date? For release and released date, we just let ct insert date type not datetime type.
If you use a wrong string it will show the error info here.
post:
{"release_date":["Date has wrong format. Use one of these formats instead: YYYY[-MM[-DD]]."]}
get:
"detail": [
        "'123213123' value has an invalid date format. It must be in YYYY-MM-DD format."
    ]

Reply:
OK, so even though the filter type is `MultiValueFilter` the format is still required to match the data somehow. Didn't know about this.

Anyway, with current filter user cannot search for records which released after, before or in between some dates. I think that's more useful than filtering by exact date. I would rather see this done in the same way as for the eol field I mentioned above.

Reply:
oh, sorry for missing the eol part of the first comment, yes, it's a very good idea.
Will add new filters here to get data.

Reply:
Commit new codes, please help to review.